components/contribution-flow/StepDetails.js

```diff
@@ -301,6 +302,7 @@ StepDetails.propTypes = {
    */
   amountOptions: PropTypes.arrayOf(PropTypes.number),
   currency: PropTypes.string.isRequired,
+  precision: PropTypes.number,
```

Comment:
No need to keep it in the propTypes right?

```suggestion
```

Reply:
Leftover by mistake.